tss-esapi-sys/regenerate-bindings.sh

```diff
@@ -48,7 +48,7 @@ cross-compile-openssl() {
 cross-compile-tpm2-tss() {
     pushd /tmp/tpm2-tss
     [ ! -f configure ] && ./bootstrap
-    ./configure --enable-fapi=no --prefix=/ --build=x86_64-pc-linux-gnu --host=$1 --target=$1 CC=$1-gcc
+    ./configure --disable-fapi --disable-policy --prefix=/ --build=x86_64-pc-linux-gnu --host=$1 --target=$1 CC=$1-gcc
```

Comment:
Hmm, I kinda hope this doesn't disable policies for ESAPI as well... 🤔

Reply:
I tried to look deep in their config files to exact what the flag did but got lost in how much I do not ... approve .. of autoconf files and syntax. But I saw that the Yocto project made some similair changes in a patch a looked for mocing to 4.0.1.